src/frontend/src/expr/expr_visitor.rs

```diff
@@ -43,14 +43,17 @@ pub trait ExprVisitor<R: Default> {
             ExprImpl::WindowFunction(inner) => self.visit_window_function(inner),
         }
     }
-    fn visit_function_call(&mut self, func_call: &FunctionCall) -> R {
+    fn visit_function_call_general(&mut self, func_call: &FunctionCall) -> R {
         func_call
             .inputs()
             .iter()
             .map(|expr| self.visit_expr(expr))
             .reduce(Self::merge)
             .unwrap_or_default()
     }
+    fn visit_function_call(&mut self, func_call: &FunctionCall) -> R {
+        self.visit_function_call_general(func_call)
+    }
```

Comment:
> Hmmm I know you want to call default impl at specialized impl, but I'd rather copy-paste instead of introducing another trait method, which looks a little bit strange 🥹

fixed